docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi9-minimal:9.0.0
```

Comment:
@pawelpaszki I've also renamed this file. Let me know if you see any issues with it.
I've also renamed the place where it is used.

Reply:
@machi1990 - I think there should be no issue

Reply:
Thanks @pawelpaszki

Reply:
Is it referenced from any jenkins pipeline / ci pipeline?

Reply:
Yes & No.

Yes - because this file is used to push changes to the version stream pipeline for auto deployment to Stage.
No-because it is not referenced directly but via the build_deploy.sh file which has been updated as well.